tests/python/gaia-ui-tests/gaiatest/gaia_test.py

```diff
@@ -331,17 +331,17 @@ def bluetooth_disable(self):
 
     @property
     def bluetooth_is_enabled(self):
-        return self.marionette.execute_script("return window.navigator.mozBluetooth.enabled")
+        return self.marionette.execute_script("return window.navigator.mozBluetooth.getAdapters()[0].state === 'enabled'")
```

Comment:
Yeah, I could make it  `getDefaultAdapter` and then query for defaultAdapter if it exist and if it doesn't exist, use the first adapter in the getAdapters() list.